setup.cfg

```diff
@@ -1,10 +1,68 @@
-[aliases]
-build = sherpa_config xspec_config build
-build_ext = sherpa_config xspec_config build_ext
-develop = sherpa_config xspec_config develop
-install = sherpa_config xspec_config install
-test = develop test
-bdist_wheel = sherpa_config bdist_wheel
+[metadata]
+name = sherpa
+author = Smithsonian Astrophysical Observatory / Chandra X-Ray Center
+author_email = cxchelp@head.cfa.harvard.edu
+license = GNU GPL v3
+license_file = COPYRIGHT
+url = http://cxc.harvard.edu/sherpa/
```

Comment:
We should probably change this to the rtd page? If nothing else we should change to the https version ;-)